### ISSUE TITLE
[stable/redis] Render error in "redis/templates/redis-master-statefulset.yaml"

### ISSUE THREAD

**Issue by msschl** [helm/charts#18201]
**Describe the bug**
Using the chart with values-production results in
```
Error: render error in "redis/templates/redis-master-statefulset.yaml": template: redis/templates/redis-master-statefulset.yaml:82:18: executing "redis/templates/redis-master-statefulset.yaml" at <eq .Values.securityContext.runAsUser 0.0>: error calling eq: incompatible types for comparison
```

**Version of Helm and Kubernetes**:
```
$ helm version
Client: &version.Version{SemVer:"v2.15.0", GitCommit:"c2440264ca6c078a06e088a838b0476d2fc14750", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.15.0", GitCommit:"c2440264ca6c078a06e088a838b0476d2fc14750", GitTreeState:"clean"}

$ kubectl version
Client Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.6", GitCommit:"96fac5cd13a5dc064f7d9f4f23030a6aeface6cc", GitTreeState:"clean", BuildDate:"2019-08-19T11:13:49Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"windows/amd64"}
Server Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.2", GitCommit:"c97fe5036ef3df2967d086711e6c0c405941e14b", GitTreeState:"clean", BuildDate:"2019-10-15T19:09:08Z", GoVersion:"go1.12.10", Compiler:"gc", Platform:"linux/amd64"}
```

**Which chart**:
[stable/redis]

**How to reproduce it** (as minimally and precisely as possible):
Use values-production to deploy the chart.

**Comment by yudaprama** [comment]
I encounter the same problem

```
helm install --name redis stable/redis
```

**Comment by alemorcuq** [comment]
Hi, @msschl.

Thank you for reporting this. It's a known issue caused by the latest release of Helm (2.15.0). You can read more about it here: https://github.com/helm/helm/issues/6708

It seems a fix it's already merged, so we just have to wait for a new release of Helm.

**Comment by msschl** [comment]
Fixed in helm version `v2.15.1`
